src/Umbraco.Core/Runtime/CoreRuntime.cs

```diff
@@ -202,7 +202,7 @@ protected virtual IFactory Boot(IRegister register, DisposableTimer timer)
                 composers.Compose();
 
                 // create the factory
-                _factory = Current.Factory = composition.CreateFactory();
```

Comment:
@bergmania  Actually this breaks running the website because when the components are initialized some components require Current.Factory to be there. I'm fixing this up in the SqlCe branch now along with other issues preventing the site from starting. I'll ping you with the rev's but should be pretty straight forward.